### PR TITLE
Added sugar logger

### PR DESCRIPTION
**Pull request by sharadregoti**

### CODE REVIEW

File: sugar_logger.go [hunk 1]
```diff
@@ -0,0 +1,24 @@
+package helpers
+
+type SugarLogger struct {
+}
+
+func (l SugarLogger) Warningf(message string, args ...interface{}) {
+	zapLogger.Sugar().Warnf(message, args)
```
Comment (YourTechBud) [inline sugar_logger.go:7]:
```suggestion
	zapLogger.Sugar().Warnf(message, args...)
```

File: sugar_logger.go [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+func (l SugarLogger) Errorf(message string, args ...interface{}) {
+	zapLogger.Sugar().Errorf(message, args)
```
Comment (YourTechBud) [inline sugar_logger.go:11]:
```suggestion
	zapLogger.Sugar().Errorf(message, args...)
```

File: sugar_logger.go [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+func (l SugarLogger) Debugf(message string, args ...interface{}) {
+	zapLogger.Sugar().Debugf(message, args)
```
Comment (YourTechBud) [inline sugar_logger.go:15]:
```suggestion
	zapLogger.Sugar().Debugf(message, args...)
```

File: sugar_logger.go [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+}
+
+func (l SugarLogger) Infof(message string, args ...interface{}) {
+	zapLogger.Sugar().Infof(message, args)
```
Comment (YourTechBud) [inline sugar_logger.go:19]:
```suggestion
	zapLogger.Sugar().Infof(message, args...)
```

**Review by YourTechBud — CHANGES_REQUESTED** [review]

**Comment by sharadregoti** [comment]
This change was introduced because we wanted badger to use the same logs of the runner.
But the logs of badger are not necessary. So we are scrapping this pr

**Pull request closed without merging**